package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "tab-split",
+  "description": "A tab-splitting extension for Firefox",
+  "version": "0.1.0",
+  "author": "Mozilla (https://mozilla.org/)",
+  "bugs": {
+    "url": "https://github.com/mozilla/tab-split/issues"
+  },
+  "dependencies": {
+  },
+  "devDependencies": {
+    "jpm": "^1.3.1"
```

Comment:
Should we be using [**jpm**](https://www.npmjs.com/package/jpm) or [**web-ext**](https://www.npmjs.com/package/web-ext)?

Reply:
It's a bootstrap extension, so only jpm can run it. It'll break eventually, but it works for now!